app/models/people/participant.rb

```diff
@@ -112,6 +117,10 @@ def method_missing(method_name, *args)
       super(method_name, *args)
     end
   end
+
+  def can_view?(object)
+    return true if object.is_a?(Participant)
```

Comment:
Ah, this where the check happens if the object is a participant. This check should be happening in the Participant signup controller.

The behavior I'd expect is that a participant can only view itself and no other participant.